server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ContestCounter.java

```diff
@@ -141,13 +145,46 @@ public static Set<Integer> pairwiseMargins(final Set<String> winners,
     contestResult.setDilutedMargin(dilutedMargin);
 
     if (ballotCount == 0L) {
-      LOGGER.error(String.format("[countContest: %s has no ballot manifests for"
-                                 + " countyIDs: %s", contestName, contestResult.countyIDs()));
+      final String dataSource = useManifests ? "ballot manifests" : "uploaded CVRs";
+      LOGGER.error(String.format("[countContest: %s has no %s for"
+                                 + " countyIDs: %s", contestName, dataSource, contestResult.countyIDs()));
     }
 
     return contestResult;
   }
 
+  /**
+   * Calculate the size of the audit universe by counting CVRs. This is the total number
+   * of CVRs in all counties that have any votes in the contest. Used for sample-size estimation.
+   * For example, if a county had 10,000 CVRs, of which only 500 contained the contest, it would
+   * contribute 10,000 to the total.
+   * Note this should *not* be used for auditing, only for sample-size estimation - the audit
+   * calculation should get this value from the manifests, not the CVRs.
+   * @param contestResult the contestResult for this contest.
```

Comment:
The return comment confuses me a bit as it is inconsistent with my interpretation of "total number of CVRs in all counties that have any votes in the contest". I would interpret that as total number of CVRs THAT have a recorded vote for the contest, rather than total number of CVRs in all counties for which the contest is relevant. ie. that CVRs without votes for the contest are also included in the count.